src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Text.Json
+{
+    public enum JsonCommentHandling : byte
+    {
+        AllowComments = (byte)1,
+        Default = (byte)0,
+        SkipComments = (byte)2,
+    }
+    public partial class JsonReaderException : System.Exception
+    {
+        public JsonReaderException(string message, long lineNumber, long lineBytePosition) { }
+        public long LineBytePosition { get { throw null; } }
+        public long LineNumber { get { throw null; } }
+    }
+    public partial struct JsonReaderOptions
+    {
+        private int _dummyPrimitive;
+        public System.Text.Json.JsonCommentHandling CommentHandling { get { throw null; } set { } }
```

Comment:
I know that comments are the big one, but are there any other options up for consideration (e.g. single-quote strings, etc.)?

Reply:
Specifically single-quote strings, no. For any other options, the bar is much higher and there needs to be a strong scenario that would push us to add it. Ideally, the 90%+ case is valid RFC-compliant JSON, and maybe certain variations can be built on top or worked around in user code. I would be interested in getting feedback on potential other options coupled with the scenario to drive it.